f8a_worker/start.py

```diff
@@ -2,8 +2,21 @@
 
 """Start the application."""
 
-from celery import Celery
+import celery
+import os
 from f8a_worker.setup_celery import init_celery, init_selinon
+from raven.contrib.celery import register_signal, register_logger_signal
+
+
+class Celery(celery.Celery):
```

Comment:
You may want to name this class something like `SentryCelery` insead of `Celery`.